doc/run/k8s/install-sqlflow.yaml

```diff
@@ -1,59 +1,49 @@
-apiVersion: apps/v1
-kind: Deployment
+apiVersion: v1
+kind: Pod
 metadata:
   name: sqlflow-server
 spec:
-  selector:
-    matchLabels:
-      app: sqlflow-server
-  template:
-    metadata:
-      labels:
-        app: sqlflow-server
-    spec:
-      containers:
-      - name: server
-        image: sqlflow/sqlflow
-        imagePullPolicy: Always
-        env:
-        - name: SQLFLOW_WORKFLOW_LOGVIEW_ENDPOINT
-          value: "http://localhost:8001"
-        - name: SQLFLOW_WORKFLOW_STEP_IMAGE
-          value: "sqlflow/sqlflow
+  containers:
+  - name: server
+    image: sqlflow/sqlflow
+    env:
+    - name: SQLFLOW_WORKFLOW_LOGVIEW_ENDPOINT
+      value: "http://localhost:8001"
+    - name: SQLFLOW_WORKFLOW_STEP_IMAGE
+      value: sqlflow:server
```

Comment:
Does this value mean the Docker image to be executed by Tekton steps?  I don't remember we have this image?

Reply:
It's used to debug, I will remove this field.